safe_control_gym/controllers/lqr/ilqr.py

```diff
@@ -275,29 +285,31 @@ def select_action(self, obs, info=None, training=False):
         Args:
             obs (ndarray): The observation at this timestep.
             info (dict): The info at this timestep.
+            training (bool): Whether the algorithm is training or evaluating.
 
         Returns:
             action (ndarray): The action chosen by the controller.
         '''
 
-        step = self.extract_step(info)
```

Comment:
Not that I am against this change, but why didn't the original implementation work? The `extract_step` function was created specifically to get this info? Are we not passing the info in every time or what?

Reply:
Will open an issue on this